ci/build.sh

```diff
@@ -4,15 +4,15 @@ goal=
 if [ "$RDBMS" == "derby" ]; then
   goal="-Pdb=derby"
 elif [ "$RDBMS" == "mariadb" ]; then
-  goal="-Pdb=mariadb"
+  goal="-Pdb=mariadb_ci"
 elif [ "$RDBMS" == "postgresql" ]; then
-  goal="-Pdb=pgsql"
+  goal="-Pdb=pgsql_ci"
 elif [ "$RDBMS" == "oracle" ]; then
-  goal="-Pdb=oracle -Dhibernate.connection.url=jdbc:oracle:thin:@localhost:1521:XE -Dhibernate.connection.username=SYSTEM -Dhibernate.connection.password=Oracle18"
+  goal="-Pdb=oracle_ci"
 elif [ "$RDBMS" == "db2" ]; then
-  goal="-Pdb=db2 -Dhibernate.connection.url=jdbc:db2://localhost:50000/orm_test -Dhibernate.connection.username=orm_test -Dhibernate.connection.password=orm_test"
+  goal="-Pdb=db2_ci"
 elif [ "$RDBMS" == "mssql" ]; then
-  goal="-Pdb=mssql -Dhibernate.connection.url=jdbc:sqlserver://localhost:1433;databaseName= -Dhibernate.connection.username=sa -Dhibernate.connection.password=hibernate_orm_test"
+  goal="-Pdb=mssql_ci"
 fi
 
 exec ./gradlew check ${goal} -Plog-test-progress=true --stacktrace
```

Comment:
if the command or arguments involve string interpolation, it would be much less error-prone to use 'eval' instead of 'exec'. I learned painful lesson on this when I need to pass JVM option with value containing quotes (e.g. **-Dhibernate.connection.username="sys as dba"**).
Seems 'exec' is mainly used for hard coded command and arguments. When dynamic string value is involved, eval is more appropriate (you can combine two together if you really wanna the benefit of exec).